js/views/search.es6.js

```diff
@@ -22,6 +23,11 @@ function Search (ops) {
 Search.prototype = $.extend({},
     Parent.prototype,
     {
+
+        _handleKeyup: function (e) {
+            this.model.set('searchText', this.$input.val());
```

Comment:
Broadcasting a model's state changes to another component is easy. 
Just use `this.model.set(attr, value)`.